### PR TITLE
Fix log_type parameter in init.pp

### PR DESCRIPTION
**Pull request by lambda123**
related to issue : #379

### CODE REVIEW

**Comment by ekohl** [comment]
To be clear, I meant the `@param log_type` description higher up in the file.

**Comment by lambda123** [comment]
seems ok now, any chance the merge to be done today ?

**Comment by ekohl** [comment]
If you can have a look at this part of it as well, then I think it should be ready:
https://github.com/theforeman/puppet-pulp/blob/108d17fa8bb684d605ebb3e41eaf7433646998f9/manifests/init.pp#L133-L134

File: manifests/init.pp [hunk 1]
```diff
@@ -131,7 +131,7 @@
 #   The desired logging level.
 #
 # @param log_type
-#   The desired logging type: Options are: syslog, console
+#   The desired logging type:
```
Comment (ekohl) [inline manifests/init.pp]: Nitpick:
```suggestion
#   The desired logging type
```

Comment (lambda123) [reply manifests/init.pp]: got it :)

**Comment by ekohl** [comment]
Looks like I forgot to check back. Thanks!

**Pull request merged**